index.html

```diff
@@ -1,183 +1,283 @@
+      <div class="card-work work">
+        <h3>Tonic</h3>
+        <ul class="proj">
+          <li><a href="#">CANOPY</a></li>
+          <li><div class="dot"></div></li>
+          <li>Back End Dev</li>
+          <li><div class="dot"></div></li>
+          <li>2015</li>
+        </ul>
+        <p class="pro">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="language">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
+      </div>
+    </div>
+    <div class="works">
+      <img src="./images/Snapshoot Portfolio-2.png" alt="Work-sample" />
+      <div class="card-work work">
+        <h3>
+          Multi-Post Stories
+        </h3>
+        <ul class="proj">
+          <li><a href="#">CANOPY</a></li>
+          <li><div class="dot"></div></li>
+          <li>Back End Dev</li>
+          <li><div class="dot"></div></li>
+          <li>2015</li>
+        </ul>
+        <p class="pro">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="language">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
+      </div>
+    </div>
+  </section>
+  <!-- ------------------------------------------------about me--------------------------------         -->
+  <section class="border">
+    <section class="about">
+      <div class="abmy">
+        <h2>
+            About <br />
+            Myself
+          </h2>
+          <p>
+            Hello I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don't hestiate to contact me.
+          </p>
+          <a href="#" class="connect">LET'S CONNECT</a>
+          <ul class="social-links">
+            <li>
+              <a href="#" class="social-icons"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#" class="social-icons"
+                ><i class="fa-brands fa-linkedin"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#" class="social-icons"
+                ><i class="fa-brands fa-facebook"></i>
+              </a>
+            </li>
+            <li>
+              <a href="#" class="social-icons"
+                ><i class="fa-brands fa-square-github"></i
+              ></a>
+            </li>
+          </ul>
+          <a href="#" class="button">Get my resume</a>
+      </div>
+      <div class="lagu">
+        <ul class="about-me">
+           <div class="down">
+            <a href="#">Languages</a>
+            <i class="fa-solid fa-chevron-right"></i>
```

Comment:
- You have used the wrong arrow icon in this section, please fix this 🙏 .Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="951" alt="etnetnr" src="https://user-images.githubusercontent.com/73607512/206395089-343e7446-b867-4e2a-b70a-9b4d596a4ba1.PNG">

Figma template:
<img width="538" alt="baetas" src="https://user-images.githubusercontent.com/73607512/206395118-0e02956f-bea9-4a8a-9626-7771da2e02bd.PNG">